examples/tutorials.jl

```diff
@@ -48,16 +48,15 @@
 
         lines!(scene, x, y2, color = :black)
         scatter!(scene, x, y2, color = :green, marker = :utriangle, markersize = 0.1)
+        # initialize the stepper and give it an output destination
+        st = Stepper(scene, "popping_demo")
```

Comment:
```suggestion
st = Stepper(scene, @replace_with_a_path)

Reply:
I'm guessing this will autofill during docs being rendered?

Reply:
yep